app/views/questions/show.html.erb

```diff
@@ -58,34 +58,36 @@
   <hr />
 
   <% comments = @node.comments_viewable_by(current_user) %>
-  <h3><span id="comment-count"><%= comments.size %></span> Comments</h3>
-  <div id="comments-list">
-  <%= javascript_include_tag "editorToolbar" %>
-  <%= javascript_include_tag "comment" %>
-    <% comments.includes([:node, :replied_comments]).order("timestamp ASC").each do |comment| %>
-      <% if comment.reply_to.nil? %>
-        <%= render :partial => "notes/comment", :locals => {:comment => comment} %>
+  <div id="legacy-editor-container" class="col-lg-10">
```

Comment:
Style fix here to shrink the width of question comment container to 75%... That's what it used to be before recent changes I can't track.

The partials for `question/show`, `wiki/show`, and `notes/show` could use some serious clean-up!